src/Sulu/Bundle/ContentBundle/Content/Types/MultipleSelect.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Content\Types;
+
+use Sulu\Component\Content\SimpleContentType;
+
+/**
+ * ContentType for a multiple select. Currently only support for checkboxes.
+ */
+class MultipleSelect extends SimpleContentType
```

Comment:
for the property param with the function `getDefaultParameter` in the content type